libs/design-system/content-templates/youtube-player/src/lib/youtube-player.component.html

```diff
@@ -0,0 +1,29 @@
+@if (hasCookiesEnabled()) {
+  <youtube-player
+    hraFeature="youtube-player"
+    hraClickEvent
+    #player
+    [videoId]="videoId()"
+    [playerVars]="playerVars()"
+    [disableCookies]="true"
+  />
+} @else {
+  <div class="youtube-thumbnail-container" hraFeature="youtube-thumbnail">
+    <a [href]="videoUrl()" target="_blank" rel="noopener noreferrer" hraFeature="watch-on-youtube" hraClickEvent>
+      <img [src]="thumbnailUrl()" [alt]="'YouTube video thumbnail for ' + videoId()" class="thumbnail-image" />
+    </a>
+    <div class="enable-cookies-message">
+      <a
+        hraFeature="enable-cookies"
+        hraClickEvent
+        (click)="onEnableCookiesClick($event)"
+        class="enable-cookies-link"
+        role="button"
```

Comment:
Don't switch the aria role. Use a button (`<button>` without `mat-button`) and style it to look like plain text?